experimental/terraform/setup.sh

```diff
@@ -75,9 +75,8 @@ echo "••••••••🔑••🔑••🔑••🔑•••••
 
 # TODO: make data generation optional
 echo "generating data…"
-# TODO: allow passing of webhook / secret as params to data generator
-export WEBHOOK=$(terraform output -raw event-handler-endpoint)
-export SECRET=$(terraform output -raw event-handler-secret)
+WEBHOOK=$(terraform output -raw event-handler-endpoint) \
+SECRET=$(terraform output -raw event-handler-secret) \
 python3 ../../data_generator/generate_data.py --vc_system=github
```

Comment:
if this is all now meant to be one command, better to intent as not to confuse readers

Reply:
Good call; fixed.